src/daemons/MetaDaemon.cpp

```diff
@@ -66,12 +66,7 @@ int main(int argc, char* argv[]) {
   // the 2nd will make the 1st failed to output log anymore
```

Comment:
Please verify if the behavior in the comment.

Reply:
In current `folly::init` and `SetUpLogging` is called after pid checking too.

Reply:
But Standalone daemon don't require this constraint even before this PR, need to modify it?

Reply:
It is ok for now. I can't clearly remember why we move `setupLogging` very close to start position of the main.